### PR TITLE
Story1 presentation

### PR DESCRIPTION
**Pull request by PlushZ**

### CODE REVIEW

File: stories/alien/level1/slides.html [hunk 1]
```diff
@@ -1,69 +1,72 @@
...
+![Sam, a small bluish unearthly creature with three eyes and six wings](images/sam.png)
 
 ???
-Meet Sam, he is an alien from the planet Gigantic. 
-His planet is very big and colorful. 
-But sometimes Sam feels a bit alone. 
-He is one of the smallest creatures on planet Gigantic. 
+- Hello, I am an alien from the planet Gigantic. 
+- My planet is very big and colorful. 
+- But sometimes I feel a bit alone.
```
Comment (teresa-m) [inline stories/alien/level1/slides.html:14]:
```suggestion
- But sometimes I feel alone. 
```

File: stories/alien/level1/slides.html [hunk 2]
```diff
@@ -1,69 +1,72 @@
...
 
 ???
-Meet Sam, he is an alien from the planet Gigantic. 
-His planet is very big and colorful. 
-But sometimes Sam feels a bit alone. 
-He is one of the smallest creatures on planet Gigantic. 
+- Hello, I am an alien from the planet Gigantic. 
+- My planet is very big and colorful. 
+- But sometimes I feel a bit alone. 
+- I am one of the smallest creatures on planet Gigantic.
```
Comment (teresa-m) [inline stories/alien/level1/slides.html:15]:
```suggestion
- I am the smallest creatures on planet Gigantic.
- At least it feels like I am.
- Sometimes I wonder what is out there in this unknown of the universe.
- Are there even smaller creatures than me. 
```

File: stories/alien/level1/slides.html [hunk 3]
```diff
@@ -1,69 +1,72 @@
...
 
-![Alt text](images/spacecraft.png)
+![A flying saucer with three portholes. Sam looks out of one porthole](images/spacecraft.png)
 
 ???
-He therefore decides to start exploring space in his spacecraft. 
-He discovers different galaxies with many different planets. 
-Most of the planets looked quite boring to Sam. 
-He therefore went further and further exploring. 
+- I therefore decide to start exploring space in his spacecraft.
```
Comment (teresa-m) [inline stories/alien/level1/slides.html:24]:
```suggestion
- Therefore, I want to explore space in my spacecraft.
- Do you want to come with me? 
```

File: stories/alien/level1/slides.html [hunk 4]
```diff
@@ -1,69 +1,72 @@
...
-![Alt text](images/spacecraft.png)
+![A flying saucer with three portholes. Sam looks out of one porthole](images/spacecraft.png)
 
 ???
-He therefore decides to start exploring space in his spacecraft. 
-He discovers different galaxies with many different planets. 
-Most of the planets looked quite boring to Sam. 
-He therefore went further and further exploring. 
+- I therefore decide to start exploring space in his spacecraft. 
+- I discover different galaxies with many different planets.
```
Comment (teresa-m) [inline stories/alien/level1/slides.html:25]:
```suggestion
- look there are different galaxies with many different planets. 
```

File: stories/alien/level1/slides.html [hunk 5]
```diff
@@ -1,69 +1,72 @@
...
 
 ???
-He therefore decides to start exploring space in his spacecraft. 
-He discovers different galaxies with many different planets. 
-Most of the planets looked quite boring to Sam. 
-He therefore went further and further exploring. 
+- I therefore decide to start exploring space in his spacecraft. 
+- I discover different galaxies with many different planets. 
+- Most of the planets look quite boring to me. 
+- I therefore go further and further exploring.
```
Comment (teresa-m) [inline stories/alien/level1/slides.html:27]:
```suggestion
- I want to go further and further exploring. 
```

File: stories/alien/level1/slides.html [hunk 6]
```diff
@@ -1,69 +1,72 @@
...
-
-![Alt text](images/earth.png)
+![Schematic representation of the globe in two colors - green and blue](images/earth.png)
 
 ???
-Until he suddenly discovered the Earth. 
-It was very special because it was blue and green, Sam’s favourite colors. 
-He decided to approach and explore the Earth. 
-As he got closer he discovered water and land. 
+- Until I suddenly discover the Earth.
```
Comment (teresa-m) [inline stories/alien/level1/slides.html:36]:
```suggestion
- Look there is an interesting planate.
- I think we discover the Earth. 
```

File: stories/alien/level1/slides.html [hunk 7]
```diff
@@ -1,69 +1,72 @@
...
+![Schematic representation of the globe in two colors - green and blue](images/earth.png)
 
 ???
-Until he suddenly discovered the Earth. 
-It was very special because it was blue and green, Sam’s favourite colors. 
-He decided to approach and explore the Earth. 
-As he got closer he discovered water and land. 
+- Until I suddenly discover the Earth. 
+- It is very special because it is blue and green, my favourite colors. 
+- I decide to approach and explore the Earth.
```
Comment (teresa-m) [inline stories/alien/level1/slides.html:38]:
```suggestion
- Let us approach and explore the Earth. 
```

File: stories/alien/level1/slides.html [hunk 8]
```diff
@@ -1,69 +1,72 @@
...
 
 ???
-Until he suddenly discovered the Earth. 
-It was very special because it was blue and green, Sam’s favourite colors. 
-He decided to approach and explore the Earth. 
-As he got closer he discovered water and land. 
+- Until I suddenly discover the Earth. 
+- It is very special because it is blue and green, my favourite colors. 
+- I decide to approach and explore the Earth. 
+- As I get closer I discover water and land.
```
Comment (teresa-m) [inline stories/alien/level1/slides.html:39]:
```suggestion
- The blue and green colors of the earth are actually water and land. 
```

File: stories/alien/level6/slides.html [hunk 9]
```diff
@@ -0,0 +1,89 @@
...
+
+### Metagenomics
+
+![There are many clusters of different sizes.](images/metagenomics3.png)
+
+<small><small>[https://www.ebi.ac.uk/about/news/feature-story/whats-a-microbiome]</small></small>
+
+???
+- Places like soil, air, water, parts of living organisms, and other places can be used as sources of samples.
+- Once samples are collected, the second step is DNA sequencing. In this step, short DNA reads can be obtained.
```
Comment (teresa-m) [inline stories/alien/level6/slides.html:48]:
```suggestion
- Once samples are collected, the second step is DNA sequencing. In this step, short DNA reads will be prepared.
```

File: stories/alien/level6/slides.html [hunk 10]
```diff
@@ -0,0 +1,89 @@
...
+---
+
+### Metagenomics
+
+![Reads that coincide with individual parts of each other and make up contigs](images/contigs.png)
+
+<small><small>[https://en.wikipedia.org/wiki/Sequence_assembly]</small></small>
+
+???
+- The first step is assembly, in which short reads are made longer by using different assembly algorithms.
```
Comment (teresa-m) [inline stories/alien/level6/slides.html:59]:
```suggestion
- Starting from this short DNA reads a metagenomics analysis can be performed. 
- The first step is assembly, in which short reads are made longer by using different assembly algorithms. 
```

File: stories/alien/level6/slides.html [hunk 11]
```diff
@@ -0,0 +1,89 @@
...
+### Metagenomics
+
+![Contigs are assigned to specific labels.](images/binning.png)
+
+<small><small>[https://angus.readthedocs.io/en/2019/recovering-rep-genomes-from-mgs.html]</small></small>
+
+???
+- Classifying (or binning) is the second step. 
+- Each contig can be assigned to the most appropriate genome.
+- At the third step, annotation, the previous step's data will be used to annotate the data.
```
Comment (teresa-m) [inline stories/alien/level6/slides.html:73]:
```suggestion
- At the third step, the so called annotation, the data from the previous steps will be used to annotate.
```

File: stories/alien/level6/slides.html [hunk 12]
```diff
@@ -0,0 +1,89 @@
...
+---
+
+### Metagenomics
+
+![Sam is in front of computer screen on which there is a pie-chart.](images/sam-and-graph.png)
+
+
+???
+- As soon as I complete the previous three steps of metagenomics I am going to proceed to the fourth step.
+- The fourth step is analysis, last but not least.
```
Comment (teresa-m) [inline stories/alien/level6/slides.html:86]:
```suggestion
- Once the first three steps of metagenomics are completed the resulting data can be analyzed. 
```

File: stories/alien/level6/slides.html [hunk 13]
```diff
@@ -0,0 +1,89 @@
...
+### Metagenomics
+
+![Sam is in front of computer screen on which there is a pie-chart.](images/sam-and-graph.png)
+
+
+???
+- As soon as I complete the previous three steps of metagenomics I am going to proceed to the fourth step.
+- The fourth step is analysis, last but not least. 
+- The experts use different tools to analyze data and draw conclusions to answer their research questions.
+- After learning about existing metagenomic analysis tools I start analyzing the data.
```
Comment (teresa-m) [inline stories/alien/level6/slides.html:88]:
```suggestion
- I want to learn more about existing metagenomic analysis tools and I also want to analyze the data.
```

File: stories/alien/level6/slides.html [hunk 14]
```diff
@@ -0,0 +1,89 @@
...
+- It is incredible since I see different colorful graphs that visually reflect information about the data.
```
Comment (teresa-m) [inline stories/alien/level6/slides.html:89]:
```suggestion
- It is fascinating what different colorful graphs that visually reflect information about the data I can see.
- Would you also want to analyze metagenome data?
```

**Review by teresa-m — COMMENTED** [review]
Thanks for the great work Polina. My comments are mostly suggestions to change the formulation of the text.

File: stories/alien/level2/slides.html [hunk 15]
```diff
@@ -0,0 +1,301 @@
...
+
+???
+- Connecting this first information that I learned I realize all life on earth is related. 
+- In other words, if I could travel back in time, I could find a common genetic ancestor for any two organisms on earth. 
+- It does not matter how different they seem. 
+- Even humans and bananas, which humans eat for breakfast, share a distant ancestor. 
+- It may seem shocking. 
+- But when comparing a human genome to a banana genome, there are so many genes that are similar within the two organisms. 
+- Although, from the outside they appear to be very different organisms. 
+- Thinking about what humans do for living and what a banana does, there's a lot of things they do in the same way, like consuming oxygen.
```
[the remark this answers is not in the document]
Comment (PlushZ) [reply stories/alien/level2/slides.html:86]: Better to remove this info

File: stories/alien/level2/slides.html [hunk 16]
```diff
@@ -0,0 +1,301 @@
...
+
+<small><small>[https://www.youtube.com/watch?v=EWjYS-DNIiQ]</small></small>
+
+???
+- I find a cell wonderful and goes back to “domains”.
+- I find out that archaea were first classified as bacterias.
+- Their appearance from the outside looks closer to bacterias.
+- However other properties are closer to the once of eukaryota.
+- Therefore, archaea form a separate domain from bacteria. 
+- Nevertheless, I do not investigate archaea further because this is a different but also fascinating story.
```
[the remark this answers is not in the document]
Comment (PlushZ) [reply stories/alien/level2/slides.html:192]: ok

File: stories/alien/level3/slides.html [hunk 17]
```diff
@@ -0,0 +1,92 @@
...
+### DNA
+
+![Schematic image of a DNA inside a chromosome, a chromosome inside a nucleus, a nucleus inside a cell](images/chromosome.png)
+
+<small><small>[https://www.yourgenome.org/video/from-dna-to-protein-flash]</small></small>
+
+???
+- After learning that DNA exists, I am eager to see this magical thing which carries all the information describing living organisms. 
+- Since I already know how tiny it is, I use my electron microscope vision. 
+- The most exciting thing I find in eukaryotic organisms.
```
[the remark this answers is not in the document]
Comment (PlushZ) [reply stories/alien/level3/slides.html:33]: Sam began to learn more about eukariotic cells, but maybe this sentence is redundant.

File: stories/alien/level3/slides.html [hunk 18]
```diff
@@ -0,0 +1,92 @@
...
+???
+- After learning that DNA exists, I am eager to see this magical thing which carries all the information describing living organisms. 
+- Since I already know how tiny it is, I use my electron microscope vision. 
+- The most exciting thing I find in eukaryotic organisms. 
+- It turns out that DNA in eukaryotes is found in structures of the cell called chromosomes. 
+- Chromosomes are thread-like structures located inside the nucleus of cells. 
+- Each chromosome packs a single molecule of deoxyribonucleic acid (DNA). 
+- For the packing processes little helpers, called protein, are needed.
+- They are part of the chromosome and situated around the packed DNA.
+
```
[the remark this answers is not in the document]
Comment (PlushZ) [reply stories/alien/level3/slides.html:39]: I agree

File: stories/alien/level2/slides.html [hunk 19]
```diff
@@ -0,0 +1,301 @@
...
+- Until you get a tree.
+- You'll be organising the collection of species into a tree. 
+- You can search for more information about species using the "magnifying glass" button, or searching more on the Internet. 
+- After you gather the information you need, build a tree by drugging species icons and then traits into the center of the screen.
+- In general, the more closely related the species the more traits they share. 
+- Deeper branch points connect species that are more distantly related and have fewer traits in common.
+- Click the button "Play" under the "Mission 1 - Training trees".
+- And then, click “Play” again under “Red, green, and gecko” and enjoy!
+- For our purpose it's enough to play only “Red, green, and gecko”.
+
```
[the remark this answers is not in the document]
Comment (PlushZ) [reply stories/alien/level2/slides.html:136]: I moved this game from slides to quiz as optional question

File: stories/alien/level2/slides.html [hunk 20]
```diff
@@ -0,0 +1,301 @@
...
+- Soil, water, air, plants, animals and humans are parts of it.
+- I find life in all kinds of forms and understands that organisms are different. 
+- But where do these differences come from?
+- Why are do small and big organisms exist?
+- And what makes all these big and tiny organisms different? 
+- I find the answer in the thought that to survive, an organism or animal must adapt well to its environment. 
+- If an animal eats insects, they must hunt for them. 
+- Their body must have tools for this, such as a beak. 
+- Other creatures, such as lizards, have a greenish and brownish body color in order to avoid falling prey to predators.
+- Therefore, diverse creatures have different colors and shapes.
```
[the remark this answers is not in the document]
Comment (PlushZ) [reply stories/alien/level2/slides.html:25]: done

File: stories/alien/level2/slides.html [hunk 21]
```diff
@@ -83,7 +83,7 @@
 - It may seem shocking. 
 - But when comparing a human genome to a banana genome, there are so many genes that are similar within the two organisms. 
 - Although, from the outside they appear to be very different organisms. 
-- Thinking about what humans do for living and what a banana does, there's a lot of things they do in the same way, like consuming oxygen. 
+- Thinking about what humans do for living and what a banana does, there's a lot of things they do in the same way.
```
Comment (teresa-m) [inline stories/alien/level2/slides.html:86]: Thinking about a human and a banana. There's a lot of things they do in the same way, like consuming oxygen.

**Pull request merged**